app/controllers/api/payment/braintree_controller.rb

```diff
@@ -29,15 +29,24 @@ def webhook
     render json: {success: true}
   end
 
+  def delete_payment_method
+    # TODO: Delete payment method on Vault, and if that's successful, destroy it locally.
```

Comment:
when it's deployed it's going to need some kind of member authentication and to me it's a no-go as a public endpoint